src/Symfony/Component/Console/Command/Command.php

```diff
@@ -29,6 +29,8 @@
  */
 class Command
 {
+    protected static $defaultName;
```

Comment:
a docblock here to help people discover how this can be leveraged, and all good on my side